util/unifyfs/src/unifyfs.c

```diff
@@ -170,20 +173,23 @@ static void parse_cmd_arguments(int argc, char** argv)
             break;
 
         case 'i':
-            printf("WARNING: stage-in not yet supported!\n");
             stage_in = strdup(optarg);
             break;
 
         case 'o':
-            printf("WARNING: stage-out not yet supported!\n");
             stage_out = strdup(optarg);
             break;
 
         case 'h':
-        default:
             usage(0);
             break;
+
+        default:
+	    printf("\n\nArgument %d is invalid!\n", argument_count);
+            usage(-EINVAL);
+            break;
         }
+        argument_count++;
     }
```

Comment:
@craigsteffen Also forgot here: if --stage-out is given, we need to verify if '--shared-dir' is given together.

Reply:
Yep.  I changed that in the commits that I'm working on.  Thanks for the cross-check.